user-impl/src/main/java/com/example/auction/user/impl/CredentialEntity.java

```diff
@@ -0,0 +1,45 @@
+package com.example.auction.user.impl;
+
+import com.lightbend.lagom.javadsl.persistence.PersistentEntity;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+public class CredentialEntity extends PersistentEntity<CredentialCommand, CredentialEvent, Optional<PCredential>> {
```

Comment:
@jroper @ignasi35 I seprated those events because. user registry save and retrieve on basis og userid but credential registry can not work on basis of that. We need to match password corresponding to username and find user id corresponding to that.